lib/init/grass.py

```diff
@@ -58,51 +58,45 @@
     print("Default locale not found, using UTF-8")  # intentionally not translatable
 
 
-def decode(bytes_, encoding=None):
+def decode(bytes_, encoding=ENCODING):
     """Decode bytes with default locale and return (unicode) string
     Adapted from lib/python/core/utils.py
```

Comment:
Should say "Adapted from lib/python/script/utils.py" (also for the other function)